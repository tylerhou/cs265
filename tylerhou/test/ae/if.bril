@main {
    cond: bool = const true;
    a: int = const 1;
    b: int = const 2;
    c: int = add a b;
    br cond .left .right;
.left:
    d: int = add a b;
    print d;
    jmp .end;
.right:
    e: int = add a b;
    print e;
    jmp .end;
.end:
}
