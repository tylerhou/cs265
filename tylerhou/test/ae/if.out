@main {
._special_entry_for_ssa:
.__b0:
  cond.0: bool = const true;
  a.0: int = const 1;
  b.0: int = const 2;
  c.0: int = add a.0 b.0;
  br cond.0 .left .right;
.left:
  d.0: int = id c.0;
  print d.0;
  jmp .end;
.right:
  e.0: int = id c.0;
  print e.0;
  jmp .end;
.end:
  e.1: int = phi __undefined e.0 .left .right;
  d.1: int = phi d.0 __undefined .left .right;
}
